backend/src/database/models/organization.ts

```diff
@@ -88,6 +88,46 @@ export default (sequelize) => {
         defaultValue: false,
         allowNull: false,
       },
+      founded: {
+        type: DataTypes.INTEGER,
+        allowNull: true,
+      },
+      industry: {
+        type: DataTypes.TEXT,
+        allowNull: true,
+      },
+      size: {
+        type: DataTypes.TEXT,
+        allowNull: true,
+      },
+      naics: {
+        type: DataTypes.ARRAY(DataTypes.JSONB),
+        allowNull: true,
+      },
+      profiles: {
+        type: DataTypes.ARRAY(DataTypes.TEXT),
+        allowNull: true,
+      },
+      headline: {
```

Comment:
What is this? And how is it different from the description? If it's semantically the same (or very similar), we should just use description.

Reply:
Could you also add comments for all the new fields that are not obvious? Like the ones where I left a comment, and naics, ticker, type...

Reply:
The `headline` field according to PDL is a piece of brief information about the company. More detailed info can be found in the company's `summary`. This is what I am using to update our `description` field. `naics` is the company's classification according to the NAIC standard. `ticker` is the company's trading symbol. And `type` is the kind of entity the company is, ex an NGO.